.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all"
```

Comment:
Should we add `bracketSpacing: false`? https://prettier.io/docs/en/options.html#bracket-spacing I prefer that style.

Reply:
We can do that, generally I changed the options to match the existing style until the diff wasn't that big anymore :wink:

Reply:
Done.